dist/wavesurfer.js

```diff
@@ -1192,14 +1192,15 @@ WaveSurfer.util.extend(WaveSurfer.MediaElement, {
      *  @param  {String}        url         path to media file
      *  @param  {HTMLElement}   container   HTML element
      *  @param  {Array}         peaks       array of peak data
+	 *  @param  {String}        preload     HTML 5 preload attribute value
      */
-    load: function (url, container, peaks) {
+    load: function (url, container, peaks, preload) {
         var my = this;
 
         var media = document.createElement(this.mediaType);
         media.controls = this.params.mediaControls;
         media.autoplay = this.params.autoplay || false;
-        media.preload = 'auto';
+	media.preload = preload == null ? 'auto' : preload;
```

Comment:
Arf, github webditor shows this as perfectly aligned. I created the indentation by copy pasting from the above line.
Well, should be corrected now.